billboard.py

```diff
@@ -65,8 +66,17 @@ class ChartEntry:
         rank: The track's position on the chart, as an int.
         isNew: Whether the track is new to the chart, as a boolean.
     """
-
-    def __init__(self, title, artist, image, peakPos, lastPos, weeks, rank, isNew):
+    def __init__(self,
+                 title,
+                 artist,
+                 image,
+                 peakPos,
+                 lastPos,
+                 weeks,
+                 rank,
+                 isNew,
+                 artistImage=None):
```

Comment:
For consistency, can you explicitly set `artistImage` to `None` in `_parseOldStylePage` instead of having a default value?

Reply:
yes, will do